catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -241,7 +240,7 @@ private QueryResponse sourceFederate(List<Source> sources, final QueryRequest qu
 
                     QueryRequest sourceQueryRequest = new QueryRequestImpl(modifiedQuery,
                             queryRequest.isEnterprise(),
-                            Collections.singleton(source.getId()),
+                            queryRequest.getSourceIds(),
```

Comment:
Should we do `new HashSet<>(queryRequest.getSourceIds())` here (like below) to ensure there are no racy issues with mutability?